composer/loggers/mlflow_logger.py

```diff
@@ -56,6 +56,7 @@ class MLFlowLogger(LoggerDestination):
         log_system_metrics (bool, optional): Whether to log system metrics. If ``True``, Mlflow will
             log system metrics (CPU/GPU/memory/network usage) during training. (default: ``True``)
         ignore_metrics (List[str], optional): A list of glob patterns for metrics to ignore when logging. (default: ``None``)
+        ignore_hyperparameters (List[str], optional): A list of glob patterns for hyperparameters to ignore when logging. (default: ``None``)
```

Comment:
Thought: what if we just use `ignore_metrics` in both places? I'm good either way though

Reply:
they have different requirements :) e.g. maybe i want to ignore all hparams (`*`) without ignoring all metrics

Reply:
it's a lil silly that we have to support both of these, but they do function as separate systems and could potentially have overlap in names, so leaving them separate is prob smart